green/result.py

```diff
@@ -737,14 +735,6 @@ def printErrors(self):
             if not self.args.no_tracebacks:
                 relevant_frames = []
                 for i, frame in enumerate(err.traceback_lines):
-                    # Python2 tracebacks containing unicode need some special handling
```

Comment:
I can see the [minimum python version is now 3.8](https://github.com/CleanCut/green/blob/main/setup.cfg#L69), so this make sense 👍